src/pages/home/report/ReportActionItem.js

```diff
@@ -623,6 +623,18 @@ function ReportActionItem(props) {
         );
     }
 
+    // For the `pay` IOU action on non-send money flow, we don't want to render anything if `isWaitingOnBankAccount` is true
+    // Otherwise, we will see two system messages informing the payee needs to add a bank account or wallet
+    if (
+        props.action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU &&
+        lodashGet(props.report, 'isWaitingOnBankAccount', false) &&
```

Comment:
This condition is just added to original PR, right?

Reply:
Yes.